### DISCUSSION TITLE
Proxied apps through wireguard tunnel

### DISCUSSION THREAD

**Discussion by luistiktok** [NginxProxyManager/nginx-proxy-manager#1505] [Q&A]
Hi. I have Proxy manager, wekan boards and wirehole (wireguard+pinhole+unbound) all docker managed with portainer.
My wekan board runs on misite.local:8080 (local wireguard ip) what i want is to redirect port 8080 to 80 but i have no success. I tried adding the local ip and port 8080 in proxy manager but nothing. What would it be the right setup to redirect ports behind my VPN environment? There is no public access, just through my VPN.

Thank you

**Comment by chaptergy** [comment]
We'll have to see the configuration you tried to help you. Under what domain is your npm instance available? You should be able to add a proxy at e.g. `wekan.proxy.local` (assuming your npm instance is available at `proxy.local`, which targets `misite.local` and port `8080`. When visiting `wekan.proxy.local` you should then be able to see your wekan board.
